modules/schema/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -2696,9 +2725,17 @@ type Attachment {
   program:        Program!
 }
 
-# Metadata for `enum AttachmentType`
-type AttachmentTypeMeta {
-  tag:        AttachmentType!
+input ObsAttachmentPropertiesInput {
+  # The description field may be unset by assigning a null value, or ignored by skipping it altogether
+  description: NonEmptyString
+
+  # The checked status can be set, or ignored by skipping it altogether
+  checked: Boolean
+}
```

Comment:
No. Although I suppose we could allow changing the file name since it is no longer used as part of the key for S3. 

I'm not allowing changing the attachment type because if the attachment is, for example, already assigned to one or more observation as a finder chart, we'd have to figure out what to do if they changed the type to MOS mask. Or maybe we'd have to check and only allow changing the type if it wasn't assigned. This seemed more complicated and I'm hoping we can get away with it.